### PR TITLE
[CHAD-8812] Updated qubino-flush-thermostat's thermostatHeatingSetpoint

### PR DESCRIPTION
**Pull request by Bhavik-T**
Updated qubino-flush-thermostat's thermostatHeatingSetpoint to -13F to 176F

### CODE REVIEW

**Comment by greens** [comment]
@bflorian about the question of units again

**Comment by lelandblue** [comment]
Hey @ctowns Can you please take a look at this its in F and not in C and I believe @bflorian said this needs to be in C. Thankyou.

**Comment by ctowns** [comment]
Converted from F to C

**Review by greens — APPROVED** [review]

**Comment by iplantemn** [comment]
@bflorian @greens updated capability configuration

File: drivers/SmartThings/zwave-thermostat/profiles/qubino-flush-thermostat.yml [hunk 1]
```diff
@@ -6,6 +6,10 @@ components:
     version: 1
   - id: thermostatHeatingSetpoint
     version: 1
+    configuration:
```
Comment (tpmanley) [inline drivers/SmartThings/zwave-thermostat/profiles/qubino-flush-thermostat.yml:9]:
```suggestion
    config:
```

File: drivers/SmartThings/zwave-thermostat/profiles/qubino-flush-thermostat.yml [hunk 2]
```diff
@@ -6,6 +6,10 @@ components:
     version: 1
   - id: thermostatHeatingSetpoint
     version: 1
+    config:
+      values:
+        - key: "temperature.value"
```
Comment (greens) [inline drivers/SmartThings/zwave-thermostat/profiles/qubino-flush-thermostat.yml:11]: yeah I think this is the wrong attribute name

**Comment by ctowns** [comment]
It looks like the thermostatHeatingSetpoint for this device was already set as a part of this PR: https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/200

In that PR. the range was set to [4, 28] with correct attribute name:
`  - id: thermostatHeatingSetpoint
    version: 1
    config:
      values:
        - key: "heatingSetpoint.value"
          range: [ 4, 28 ]`
          
In this patch however, it is being set to [-25, 80]. Which of these numbers should be considered "correct"? [CHAD-8812](https://smartthings.atlassian.net/browse/CHAD-8812) calls out that the temperature range should be [-25, 80], but I am wondering if that is a misunderstanding. According to the [qubino thermostat technical specification](https://qubino.com/products/flush-onoff-thermostat/flush-on-off-thermostat-technical-specifications2/), the temperature sensor range is [-25, 80] C but the "operational range" is only [-10, 40] C. I am not sure where the [4, 28] value from the other PR came from, but which of these values is correct?

**Comment by greens** [comment]
@ctowns I'm guessing the sensor can detect -25,80, but the setpoints are restricted to -10,40, so -10,40 is probably what we want.

My guess is that Bhavik went with a general range from the DTH side of things and applied it to all thermostats, and some of them have specific values and that's what this PR is about.

**Review by greens — APPROVED** [review]

**Pull request merged**